source/administration/config-settings.rst

```diff
@@ -2313,6 +2313,25 @@ If the "No encryption" option is selected it is highly recommended that the AD/L
 | This feature's ``config.json`` setting is ``"ConnectionSecurity": ""`` with options ``""``, ``"TLS"``, and ``"STARTTLS"``.                                           |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Private Key
```

Comment:
Both of the tables are broken.

Reply:
Fixed - didn't click that the edits would mess up the tables.